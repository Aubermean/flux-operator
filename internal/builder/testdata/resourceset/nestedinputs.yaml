apiVersion: fluxcd.controlplane.io/v1
kind: ResourceSet
metadata:
  name: app1
  namespace: apps
spec:
  inputs:
  - foo:
      bar:
        k1: v1
        k2: v2
  resources:
  - apiVersion: v1
    kind: ConfigMap
    metadata:
      name: configmap1
    data: << inputs.foo.bar | toYaml | nindent 2 >>
