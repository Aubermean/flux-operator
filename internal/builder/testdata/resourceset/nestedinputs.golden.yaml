apiVersion: v1
data:
  k1: v1
  k2: v2
kind: ConfigMap
metadata:
  name: configmap1
---
